cryptus-app/components/market/market.tsx

```diff
@@ -94,16 +114,41 @@ export default function MarketOverview(props: market_overview_props) {
   };
 
   const close_card = () => {
-    console.log("close card");
     set_show_card(false);
   };
 
   const open_card = (i) => {
-    console.log("open card");
     set_card_index(i);
     set_show_card(true);
   };
 
+  const close_all = () => {
+    console.log("close_all");
+
+    set_show_card(false);
+    set_show_settings(false);
+    set_show_wallet_manager(false);
+  };
+
+  const close_wallet = () => {
+    set_show_wallet_manager(false);
+  };
+
+  const close_support = () => {
+    set_show_support(false);
+  };
+
+  const open_settings = () => {
+    set_show_settings(true);
+  };
+
+  const open_support = () => {
+    set_show_support(true);
+  };
+
+  const open_wallet_manager = () => {
+    set_show_wallet_manager(true);
+  };
```

Comment:
all the cards polute the market (and the profile) so much. I wonder if there is a way to have that logic elsewhere. What do you think @GuilaumeBrassard @turcottep

Reply:
I guess on peut import d'un autre fichier dans lib